src/main/resources/swagger.api/circulation-item.yaml

```diff
@@ -35,13 +35,13 @@ paths:
       operationId: retrieveCirculationItemById
       responses:
         '200':
-          description: ""
+          description: Circulation Item updated successfully
```

Comment:
Description should be something like Circulation Item retrieved successfully for get API.

Reply:
done